hx_lti_initializer/templatetags/hx_lti_initializer_extras.py

```diff
@@ -51,8 +51,8 @@ def format_tags(tagslist):
 	'''
 		Pretty-prints list of tags
 	'''
-	if tagslist == ['']:
-		return "None"
+	if tagslist == []:
```

Comment:
This is just checking that it's an empty list, right? You could eliminate the conditional, because if it's an empty list, you still get the desired result according to my trusty REPL:

```python
>>> ', '.join([])
''
```